### PR TITLE
Change emergency kit format

### PR DESCRIPTION
**Pull request by bramkragten**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/config/backup/ha-config-backup-settings.ts [hunk 1]
```diff
@@ -141,9 +141,9 @@ class HaConfigBackupSettings extends LitElement {
             <div class="card-content">
               <p>
                 Keep this encryption key in a safe place, as you will need it to
-                access your backup, allowing it to be restored. Either record
-                the characters below or download them as an emergency kit file.
-                Encryption keeps your backups private and secure.
+                access your backup, allowing it to be restored. Download them as
+                an emergency kit file and store it somewhere save. Encryption
```
Comment (mbo18) [inline src/panels/config/backup/ha-config-backup-settings.ts:145]: safe instead of save?

Comment (bramkragten) [inline src/panels/config/backup/ha-config-backup-settings.ts:145]:
```suggestion
                an emergency kit file and store it somewhere safe. Encryption
```

File: src/data/backup.ts [hunk 2]
```diff
@@ -288,6 +293,39 @@ export const generateEncryptionKey = () => {
...
+  "data:text/plain;charset=utf-8," +
+  encodeURIComponent(`Home Assistant Backup Emergency Kit
+
+This emergency kit contains your backup encryption key. You need this key
+to be able to restore your Home Assistant backups.
+
+Date: ${formatDateTime(new Date(), hass.locale, hass.config)}
+Instance: ${hass.config.location_name}
+URL: ${hass.auth.data.hassUrl}
+Encryption key: ${encryptionKey}
```
Comment (ludeeus) [inline src/data/backup.ts:309]: Put this on a new line so there are no spaces before that could be copied in as part of the key.

File: src/data/backup.ts [hunk 3]
```diff
@@ -288,6 +293,39 @@ export const generateEncryptionKey = () => {
...
+
+For more information: https://www.home-assistant.io/use-backup-emergency-kit`);
```
Comment (ludeeus) [inline src/data/backup.ts:311]: We should use the more-info structure for these.

Comment (balloob) [inline src/data/backup.ts:311]: Can we change url to be /more-info/backup-emergency-kit

**Review by balloob — DISMISSED** [review]

**Review by balloob — APPROVED** [review]

**Review by MindFreeze — APPROVED** [review]

**Pull request merged**